Doc/whatsnew/3.14.rst

```diff
@@ -761,6 +761,14 @@ Other language changes
   ASCII :class:`bytes` and :term:`bytes-like objects <bytes-like object>`.
   (Contributed by Daniel Pope in :gh:`129349`.)
 
+* :func:`str`, :func:`repr` and :func:`ascii` now always return an instance
+  of :class:`str`, even if the special methods ``__str__()`` or ``__repr__()``
+  return an instance of a subclass of :class:`!str`.
+  :func:`bytes` now always returns an instance of :class:`bytes`, even
+  if the special method ``__bytes__()`` returns an instance of a subclass of
+  :class:`!bytes`.
+  (Contributed by Serhiy Storchaka in :gh:`104231`.)
```

Comment:
Oh wait, you should now move this text (and the addition below) to Doc/whatsnew/3.15.rst.